src/Humanizer.Tests/Localisation/DefaultFormatterTests.cs

```diff
@@ -20,5 +21,25 @@ public void TimeSpanHumanizeThrowsExceptionForTimeUnitsLargerThanWeek(TimeUnit t
         {
             Assert.Throws<ArgumentOutOfRangeException>(() => new DefaultFormatter(CultureInfo.InvariantCulture.Name).TimeSpanHumanize(timeUnit, unit));
         }
+
+        [Fact]
+        public void Issue_392_A_collection_formatter_for_the_current_culture_has_not_been_implemented_yet()
+        {
+            var originalCulture = Thread.CurrentThread.CurrentCulture;
+            var originalUiCulture = Thread.CurrentThread.CurrentUICulture;
+
+            Thread.CurrentThread.CurrentCulture = CultureInfo.CreateSpecificCulture("es");
```

Comment:
I think we need to use different culture, because "es" culture is going to be implemented some day, and the test will break.